scripts/hid_configurator/led_stream.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import random
+import time
+
+from configurator_core import fetch_free_steps_buffer_info, led_send_single_step
+from configurator_core import Step
```

Comment:
why sedond line import from the same package? Do we have 80 characters line break in python scripts also?

Reply:
This was done to improve readability of the code (similar imports are grouped)